.github/workflows/esti.yaml

```diff
@@ -27,7 +27,10 @@ jobs:
         run: |
           echo "Enable next jobs based on secrets existence: ${{ env.ENABLE_NEXT_JOBS != '' }}"
           echo "secretsavailable=${{ env.ENABLE_NEXT_JOBS != '' }}" >> $GITHUB_OUTPUT
-      
+  
```

Comment:
So much [nothingness](https://this-page-intentionally-left-blank.org/) (please don't)!

[See 2nd page of this doc](https://ptgmedia.pearsoncmg.com/images/9780132101301/samplepages/0132101300.pdf).

Reply:
oops 😅